test/unit/test_disk_tensor.py

```diff
@@ -145,5 +146,12 @@ def test_reshape(self):
     helper_test_disk_tensor("dt5", [1,2,3,4,5], lambda x: x.reshape((1,5)))
     helper_test_disk_tensor("dt6", [1,2,3,4], lambda x: x.reshape((2,2)))
 
+  @unittest.skipIf(Device.DEFAULT != "METAL", "only metal")
+  def test_metal(self):
+    pathlib.Path(temp("dt7")).write_bytes(struct.pack("<III", 1, 2, 3)) # metal is little-endian
+    t1 = Tensor.empty(3, device=f"disk:{temp('dt7')}", dtype=dtypes.uint32)
+    t2 = Tensor([4, 5, 6], dtype=dtypes.uint32)
+    np.testing.assert_allclose((t1 + t2).realize().numpy(), np.array([5, 7, 9], dtype=np.uint32))
```

Comment:
@imaolo i saw some of your PRs on this, i got this test to pass. what do you think of the approach?